scripts/deploy-ha.sh

```
@@ -0,0 +1,422 @@
+        exit 1
+    fi
+    
+    log "Scaling $service to $replicas replicas..."
+    docker-compose -f "$COMPOSE_FILE" up -d --scale "$service=$replicas"
+}
+
+# Backup function
+backup() {
+    log "Creating backup..."
+    
+    local backup_dir="$PROJECT_ROOT/backups/$(date +%Y%m%d_%H%M%S)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)